README.md

```diff
@@ -202,6 +202,16 @@ const coolImage = document.querySelector('.image-to-load-first')
 observer.triggerLoad(coolImage);
 ```
 
+### Large image improvment
+
+Sometimes image loading takes a long time. For this case, you can add a placeholder color:
+
+```html
+<img class="lozad" data-placeholder="red" data-src="image.png" />
+```
+
+Lozad set the background of `img` element and users will see the fallback.
```

Comment:
Set a placeholder background color of `img` element and users will see the fallback till the image loads.